xpath-cheatsheet.md

```diff
@@ -9,5 +9,7 @@
 |*|Any element|
 |*[@attribute='value']|Any element with a certain value for an attribute|
 |*[contains(@attribute, 'value')]|Any element which has an attribute that contains 'value'|
+|*[contains(text(), 'this text')]|Any element, that contains exactly 'this text', precisely, including whitespaces.|
+|*[contains(normalize-space(.), 'this text')]|Any element, that contains 'this text', stripped of leading or trailing whitespaces, or sequences of whitespaces inside it.|
```

Comment:
A short example could be added here, e.g
`"     this\n            text     "` vs `"   this text\n"`. (GitHub UI seems to strip additional whitespace from inline code)